README.md

```diff
@@ -275,6 +280,176 @@ Reads if device geolocation is activated or not (read-only).
 console.log(pk.hasGeolocation); // true or false
 ```
 
+### `pk.patch.list()`
+
+List, filter and paginate patches.
+⚠️ Restricted to **private** API keys, **do not expose it to the browser**.
+
+```js
+// get all patches, paginated
+pk.patch.list().then((res) => {
+  console.log(res.results);
+});
+
+// filter and paginate patches
+pk.patch.list({
+  query: 'angeles',
+  countries: ['us'],
+  types: ['street'],
+  status: 'approved',
+  maxResults: 10,
+  offset: 0,
+}).then((res) => {
+  console.log(res.results);
+});
+```
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `query` | `string?` | Search query terms. |
+| `opts` | `key-value mapping` (optional) | Search options. |
+| `opts.status` | `('all' | 'pending' | 'approved')?` | Publication status. |
+| `opts.countries` | `string[]?` | Countries filter, array of [two-letter ISO](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country codes. |
+| `opts.types` | `string[]?` | Types filter, array of accepted values: `street`, `city`, `airport`, `bus`, `train`, `townhall`, `tourism`. Prepend `-` to omit a type like `['-bus']`. Unset to return all. |
+| `opts.maxResults` | `number?` | Maximum number of results to return. |
+| `opts.offset` | `number?` | Paginate results starting from the offset. |
+
+### `pk.patch.create()`
+
+Add a missing record or fix an existing one.
+⚠️ Restricted to **private** API keys, **do not expose it to the browser**.
+
+```js
+// Adding a missing record
+const record = {
+  type: 'street',
+  name: 'Example street',
+  city: 'Los Angeles',
+  county: 'Los Angeles',
+  administrative: 'California',
+  country: 'United States of America',
+  countrycode: 'us',
+  coordinates: '33.9955095,-118.472482',
+  zipcode: [90291],
+  population: 3849000, // optional
+};
+pk.patch.create(record, { status: 'approved' }).then((record) => {
+  console.log(record);
+});
+
+// Fixing an existing record
+pk.patch.create(
+  { population: 3849000 },
+  originalRecord, // original record from `pk.search` or `pk.reverse`
+  { status: 'approved' }
+).then((record) => {
+  console.log(record);
+});
+```
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `update` | `key-value mapping` | Full record if adding, only updated fields if fixing. |
```

Comment:
Do you mean `record`?

Reply:
Well, in `patch.create` it's either a "record" (all fields) or an "update" (partial), depending if `origin` is defined. I have to choose only one name for both `add` and `fix` in `patch.create` for the parameters markdown table and for JSDoc. So I prefer calling them all "update" to be consistent with `patch.update` too.